flowcraft/generator/process_details.py

```diff
@@ -71,6 +71,22 @@ def procs_dict_parser(procs_dict):
                     arg_msg = "None"
                 else:
                     arg_msg = ", ".join(dict_proc_info[info])
+            elif isinstance(dict_proc_info[info], dict):
```

Comment:
It's not clear to me what the dict_proc_info is, even with the doc string. Moreover, it seems dangerous to assume `dict` instances will always be directives.

Reply:
We can check for directives instead of assuming it is a dict, may be it is safer given that the parsing below this is very specific for directives.